js/build.js

```diff
@@ -243,9 +243,24 @@ Fliplet.Widget.instance('login-ds', function(data) {
           _this.find('.btn-label').removeClass('hidden');
           _this.find('.loader').removeClass('show');
           _this.parents('.form-btns').find('.login-error').html('Your email or password don\'t match. Please try again.').removeClass('hidden');
-          Fliplet.UI.Toast.error(error, {
-            message: 'Login error'
-          });
+          Fliplet.UI.Toast({
```

Comment:
@tonytlwu this doesn't look very polished to me. Can you provide some guidance on how it should be optimised or just confirm if this is fine?

Reply:
@YaroslavOvdii If we needed a custom duration, we could just make the duration for `Fliplet.UI.Toast.error()` customisable. However, I've left a comment https://github.com/Fliplet/fliplet-studio/issues/5787#issuecomment-603816355 for CS to confirm the behaviour first.